f2/src/ThankYouPage.js

```diff
@@ -29,6 +29,22 @@ export const ThankYouPage = () => {
 
   const [, dispatch] = useStateValue()
 
+  // Message displayed on Thank you Page
+  let reportId = state.formData.reportId
+  let thankYouMessage
+
+  if (!reportId || reportId === '') {
+    thankYouMessage = <Trans id="thankYouPage.reportSubmission" />
```

Comment:
I didn't know you could do this. Nice! 🎉

Reply:
Me either! Lots of googling :p :p